### ISSUE TITLE
Fix test_convolve_cube in gammapy/cube/tests/test_core.py

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#764]
PyCharm tells me that there's a bug in this test:
https://github.com/gammapy/gammapy/blame/master/gammapy/cube/tests/test_core.py#L273
The Quantity import is missing.

The fact that travis-ci passes means that the test never runs there, i.e. there's some problem with the
```
@requires_dependency('scipy.interpolate.RegularGridInterpolator')
@requires_dependency('reproject')
```
Probably it doesn't work for strings like "scipy.interpolate.RegularGridInterpolator".

@adonath - Do you have time to fix this tomorrow or should I do it?

**Comment by adonath** [comment]
I can fix the issue , I'm working on `SkyCube` anyway...

**Comment by cdeil** [comment]
@adonath - Thanks! Assigning to you ...

**Comment by adonath** [comment]
Fixed by #766
